server/models/Message.js

```diff
@@ -4,7 +4,29 @@ class Message extends Model {}
 
 Message.init(
     {
-        id:{},
+        id: {
+            type: DataTypes.INTEGER,
+            autoIncrement: true,
+            primaryKey: true,
+          },
+          body: {
+            type: DataTypes.STRING,
+            allowNull: false,
+            validate: {
+              len: [1],
+            },
+          },
+        //   sender
```

Comment:
this is part of the confusion I have with messages -- I need to check out your project two code in regards to this.